x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStore.java

```diff
@@ -155,6 +155,25 @@
                                 RoleDescriptor.IndicesPrivileges.builder()
                                     .indices(".logs-endpoint.diagnostic.collection-*")
                                     .privileges("read").build(),
+                                // Fleet Server indices Kibana read and write to these indices to manage Elastic Agents
+                                RoleDescriptor.IndicesPrivileges.builder()
+                                    .indices(".fleet-agents")
+                                    .privileges("read", "write").build(),
```

Comment:
Does Kibana need write permissions for the agents?

Reply:
Yes right now if we want to reassign an agent to a new policy we do it by writing to `.fleet-agents`

Reply:
Would be nice to use the doc line 158 to also mention why a specific access is needed.

Reply:
I was hoping for a bit more details and have comments on all the lines. For example that write permissions are need to force unenroll etc.

Reply:
Just updated with more a per indice comment 👍